recipes/server.rb

```diff
@@ -66,3 +66,13 @@
 sysctl_param 'vm.swappiness' do
   value 0
 end
+
+cookbook_file '/usr/local/bin/mysql-accounting' do
+  source 'mysql-accounting'
+  mode '0755'
+end
```

Comment:
I recommend you move it to ``/usr/local/sbin`` then. I'm still wondering if we want this in a ``$PATH`` or elsewhere in the system. Perhaps using ``/usr/local/libexec`` might be a better location. I highly doubt we'll want to run this manually that often.